exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/PlannerSettings.java

```diff
@@ -83,6 +83,8 @@
 
   public static final String TYPE_INFERENCE_KEY = "planner.enable_type_inference";
   public static final BooleanValidator TYPE_INFERENCE = new BooleanValidator(TYPE_INFERENCE_KEY, true);
+  public static final OptionValidator IN_SUBQUERY_THRESHOLD =
+          new PositiveLongValidator("planner.in_subquery_threshold", Integer.MAX_VALUE, 20);
```

Comment:
you should add a comment here for the default value of 20 is chosen to match the default Calcite value.